PrimerSDK.podspec

```diff
@@ -10,8 +10,7 @@ Pod::Spec.new do |s|
     s.author       = { "Primer" => "dx@primer.io" }
     s.source       = { :git => "https://github.com/primer-io/primer-sdk-ios.git", :tag => "#{s.version}" }
 
-    s.swift_version = "4.2"
-    s.ios.deployment_target = "10.0"
+    s.ios.deployment_target = '13.1'
```

Comment:
Is this bump to 13.1 required?

Reply:
Yes, it is a minimum version for Nol to be able to work.